src/components/adherence/summary/AdherenceAlerts.tsx

```diff
@@ -1,39 +1,43 @@
 import {useAdherenceAlerts, useUpdateAdherenceAlerts} from '@components/studies/adherenceHooks'
-import Loader from '@components/widgets/Loader'
 import ConfirmationDialog from '@components/widgets/ConfirmationDialog'
+import Loader from '@components/widgets/Loader'
```

Comment:
ignore import changes. I just have a setting to sort them on save. In settings.json in vscode I have '"editor.codeActionsOnSave": {
        "source.organizeImports": true,
    },